modules/ROOT/pages/policy-mule4-detokenization.adoc

```diff
@@ -6,7 +6,7 @@ endif::[]
 [%autowidth.spread,cols="a,a"]
```

Comment:
I have been editing this PR for term logic:

Tokenization replaces a value with a token, to protect that value.
Detokenization returns the tokenized value to its original value.

Some of your original text implies just the opposite, so please very carefully evaluate the suggested rewordings to ensure that I have not misunderstood and that your intended meaning is reflected.

Reply:
Noted and changed accordingly!